### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by misherwa**
Adding an issue template file as mentioned in Design Document 2 that the issue template is required. 

The template includes a description, link, optical screenshots, and additional context sections.

### CODE REVIEW

**Comment by thecatfour** [comment]
I think "Screenshots (Optional)" should be renamed to "Screenshots/Example (Optional)" to emphasize that a small example of a bug should be showcased.

**Comment by misherwa** [comment]
Thanks for your input.

I made changes to line 16 by adding the "/Example" and it looks like this:
q

**Comment by barrioskevin** [comment]
was it possible to make the issue template on your own fork and then make the pull request? right now it looks like you did your work here on the main repository which should **not** be the case. Unless of course there is something I'm missing and you needed to work directly on this repo. 

The issue template looks good although I suggest we add a "potential solution" section or at least a section where we can write in what file or files would need to be changed in order for the fix. Also maybe like a frontend, backend, or both category

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+about: Create a useable template format for guidance
+title: '[ISSUES]'
+labels: 'Issue'
+assignees: ''
+
+---
+
+**Description of the issue**
+Describe the issue and share important details such as the desired solution, error messages, and where the issue originates.
+
```
Comment (barrioskevin) [inline .github/ISSUE_TEMPLATE/issue_template.md:15]: **Files that need to be updated** (something along those lines)
Describe what files or directories that would need to be changed / updated to fix issue.
(Im saying this should be different then the description.)

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name: Issue_Template
+about: Create a useable template format for guidance
+title: '[ISSUES]'
+labels: 'Issue'
+assignees: ''
+
+---
+
+**Description of the issue**
+Describe the issue and share important details such as the desired solution, error messages, and where the issue originates.
```
Comment (barrioskevin) [inline .github/ISSUE_TEMPLATE/issue_template.md]: Describe the issue and share important details such as domain (frontend, backend, or both), error messages, and where the issue originates or is seen. (just my suggestion not forcing you to make any changes).

**Comment by misherwa** [comment]
> was it possible to make the issue template on your own fork and then make the pull request? right now it looks like you did your work here on the main repository which should **not** be the case. Unless of course there is something I'm missing and you needed to work directly on this repo.
> 
> The issue template looks good although I suggest we add a "potential solution" section or at least a section where we can write in what file or files would need to be changed in order for the fix. Also maybe like a frontend, backend, or both category

In our main repository, if you go to settings and scroll down to features. This is what it looks like:
q

If I try creating an issues template in my fork the same way by going into settings, I get this:
q2

So there isn't a "Set up templates" button when I try doing it on my fork. I could try creating a .md file on my fork and then make a PR for it, and it should work. Do you want me to try doing that?

**Comment by barrioskevin** [comment]
I think it's kinda too late now. we should stick with how you have it now but once the PR gets closed delete the added branch and in the future we stick to editing our forks. I believe you need to enable  the feature in your fork in order to see the button or creating the md template file in the appropriate directory should also work. Sorry I can't suggest a proper solution im not all familiar with the whole template thing. In regards to the actual template I think we should include a domain section where we specify (frontend, backend, or both).

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 2 above)
Comment (misherwa) [reply .github/ISSUE_TEMPLATE/issue_template.md]: I think you're right about adding details like what domain the issue is from.

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 1 above)
Comment (misherwa) [reply .github/ISSUE_TEMPLATE/issue_template.md:15]: Gotcha, I'll add this

**Comment by misherwa** [comment]
> I think it's kinda too late now. we should stick with how you have it now but once the PR gets closed delete the added branch and in the future we stick to editing our forks. I believe you need to enable the feature in your fork in order to see the button or creating the md template file in the appropriate directory should also work. Sorry I can't suggest a proper solution im not all familiar with the whole template thing. In regards to the actual template I think we should include a domain section where we specify (frontend, backend, or both).


No worries, I'll try to make sure I'm doing things in my fork instead of the main repository, and I'll delete the branch once this PR gets pulled.
 
I just updated the template to include the domain affected in the "Description of the issue":
q

And the added "Files that need to be updated" requirement:
q2

**Comment by barrioskevin** [comment]
Looks good to me!

**Comment by nnguyen144** [comment]
I think this issue template will help us organize our issues and connect our pull request with them. However I don't think the 
>**Link**
>Include a link to the correlated PR number, or any resources for greater context.

is necessary (at least for the PR part) since we can already link a pull request with an issue through GitHub.

**Review by leed85 — APPROVED** [review]
looks good, the only thing maybe is to add the goal of what the issue was trying to fix to make it more clear for others to make suggestions

**Pull request merged**